v3/4pplet/waffling80_rev_b_ansi.json

```diff
@@ -0,0 +1,291 @@
+    "0,0\n\n\n0,0",
+    {
+      "x": 1,
+      "c": "#cccccc"
+    },
+    "0,1\n\n\n0,0",
+    "1,1\n\n\n0,0",
+    "0,2\n\n\n0,0",
+    "1,2\n\n\n0,0",
+    {
+      "x": 0.5,
+      "c": "#aaaaaa"
+    },
+    "0,3\n\n\n0,0",
+    "1,3\n\n\n0,0",
+    "0,4\n\n\n0,0",
+    "1,4\n\n\n0,0",
+    {
+      "x": 0.5,
+      "c": "#cccccc"
+    },
+    "0,5\n\n\n0,0",
+    "1,5\n\n\n0,0",
+    "0,6\n\n\n0,0",
+    "1,6\n\n\n0,0",
+    {
+      "x": 0.25,
+      "c": "#aaaaaa"
+    },
+    "0,7\n\n\n0,0",
+    "1,7\n\n\n0,0",
+    "3,7\n\n\n0,0"
+  ],
+  [
+    {
+      "y": 0.25,
+      "x": 2.75,
+      "c": "#cccccc"
+    },
+    "2,0",
+    "3,0",
+    "2,1",
+    "3,1",
+    "2,2",
+    "3,2",
+    "2,3",
+    "3,3",
+    "2,4",
+    "3,4",
+    "2,5",
+    "3,5",
+    "2,6",
+    {
+      "c": "#aaaaaa",
+      "w": 2
+    },
+    "6,7\n\n\n1,0",
+    {
+      "x": 0.25
+    },
+    "2,7",
+    "5,7",
+    "9,7",
+    {
+      "x": 1,
+      "c": "#cccccc"
+    },
+    "3,6\n\n\n1,1",
+    "6,7\n\n\n1,1"
+  ],
+  [
+    {
+      "x": 2.75,
+      "c": "#aaaaaa",
+      "w": 1.5
+    },
+    "4,0",
+    {
+      "c": "#cccccc"
+    },
+    "5,0",
+    "4,1",
+    "5,1",
+    "4,2",
+    "5,2",
+    "4,3",
+    "5,3",
+    "4,4",
+    "5,4",
+    "4,5",
+    "5,5",
+    "4,6",
+    {
+      "w": 1.5
+    },
+    "5,6",
+    {
+      "x": 0.25,
+      "c": "#aaaaaa"
+    },
+    "4,7",
+    "7,7",
+    "11,7"
+  ],
+  [
+    {
+      "x": 2.75,
+      "w": 1.75
+    },
+    "6,0",
+    {
+      "c": "#cccccc"
+    },
+    "7,0",
+    "6,1",
+    "7,1",
+    "6,2",
+    "7,2",
+    "6,3",
+    "7,3",
+    "6,4",
+    "7,4",
+    "6,5",
+    "7,5",
+    {
+      "c": "#aaaaaa",
+      "w": 2.25
+    },
+    "7,6"
+  ],
+  [
+    {
+      "x": 2.75,
+      "w": 2.25
+    },
+    "8,0",
+    {
+      "c": "#cccccc"
+    },
+    "8,1",
+    "9,1",
+    "8,2",
+    "9,2",
+    "8,3",
+    "9,3",
+    "8,4",
+    "9,4",
+    "8,5",
+    "9,5",
+    {
+      "c": "#aaaaaa",
+      "w": 2.75
+    },
+    "8,6\n\n\n2,0",
+    {
+      "x": 1.25
+    },
+    "8,7",
+    {
+      "x": 1.25,
+      "c": "#cccccc",
+      "w": 1.75
+    },
+    "8,6\n\n\n2,1",
+    "9,6\n\n\n2,1"
+  ],
+  [
+    {
+      "x": 2.75,
+      "c": "#aaaaaa",
+      "w": 1.5
+    },
+    "10,0",
+    "11,1\n\n\n3,0",
+    {
+      "w": 1.5
+    },
+    "10,1",
+    {
+      "c": "#cccccc",
+      "w": 7
+    },
+    "10,3\n\n\n4,0",
+    {
+      "c": "#aaaaaa",
+      "w": 1.5
+    },
+    "11,4",
+    "10,5\n\n\n3,0",
+    {
+      "w": 1.5
+    },
+    "11,5",
+    {
+      "x": 0.25
+    },
+    "10,6",
+    "11,6",
+    "10,7"
+  ],
+  [
+    {
+      "x": 4.25,
+      "d": true
+    },
+    "11,1\n\n\n3,1",
+    {
+      "x": 1.5,
+      "c": "#cccccc",
+      "w": 3
+    },
+    "11,2\n\n\n4,1",
+    "10,3\n\n\n4,1",
+    {
+      "w": 3
+    },
+    "11,3\n\n\n4,1",
+    {
+      "x": 1.5,
+      "c": "#aaaaaa",
+      "d": true
+    },
+    "10,5\n\n\n3,1"
+  ]
+]
+  }
+}
```

Comment:
Just a pre-emptive file formatting so that once QMK merge goes through I can merge this right away.
```suggestion
{
  "name": "waffling80 Rev B ANSI HS",
  "vendorId": "0x4444",
  "productId": "0x0010",
  "matrix": {
    "rows": 12,
    "cols": 8
  },
  "keycodes": [
    "qmk_lighting"
  ],
  "menus": [
    "qmk_rgblight"
  ],
  "layouts": {
    "labels": [
      "F13",
      "Split Backspace",
      "Split Right Shift",
      "WKL",
      "Split Space"
    ],
    "keymap": [
      [
        {
          "x": 2.75,
          "c": "#aaaaaa"
        },
        "0,0\n\n\n0,1",
        {
          "x": 0.25,
          "c": "#cccccc"
        },
        "1,0\n\n\n0,1",
        "0,1\n\n\n0,1",
        "1,1\n\n\n0,1",
        "0,2\n\n\n0,1",
        {
          "x": 0.25,
          "c": "#aaaaaa"
        },
        "1,2\n\n\n0,1",
        "0,3\n\n\n0,1",
        "1,3\n\n\n0,1",
        "0,4\n\n\n0,1",
        {
          "x": 0.25,
          "c": "#cccccc"
        },
        "1,4\n\n\n0,1",
        "0,5\n\n\n0,1",
        "1,5\n\n\n0,1",
        "0,6\n\n\n0,1",
        {
          "x": 0.25
        },
        "1,6\n\n\n0,1",
        {
          "x": 0.25,
          "c": "#aaaaaa"
        },
        "0,7\n\n\n0,1",
        "1,7\n\n\n0,1",
        "3,7\n\n\n0,1"
      ],
      [
        {
          "y": 0.5,
          "x": 2.75
        },
        "0,0\n\n\n0,0",
        {
          "x": 1,
          "c": "#cccccc"
        },
        "0,1\n\n\n0,0",
        "1,1\n\n\n0,0",
        "0,2\n\n\n0,0",
        "1,2\n\n\n0,0",
        {
          "x": 0.5,
          "c": "#aaaaaa"
        },
        "0,3\n\n\n0,0",
        "1,3\n\n\n0,0",
        "0,4\n\n\n0,0",
        "1,4\n\n\n0,0",
        {
          "x": 0.5,
          "c": "#cccccc"
        },
        "0,5\n\n\n0,0",
        "1,5\n\n\n0,0",
        "0,6\n\n\n0,0",
        "1,6\n\n\n0,0",
        {
          "x": 0.25,
          "c": "#aaaaaa"
        },
        "0,7\n\n\n0,0",
        "1,7\n\n\n0,0",
        "3,7\n\n\n0,0"
      ],
      [
        {
          "y": 0.25,
          "x": 2.75,
          "c": "#cccccc"
        },
        "2,0",
        "3,0",
        "2,1",
        "3,1",
        "2,2",
        "3,2",
        "2,3",
        "3,3",
        "2,4",
        "3,4",
        "2,5",
        "3,5",
        "2,6",
        {
          "c": "#aaaaaa",
          "w": 2
        },
        "6,7\n\n\n1,0",
        {
          "x": 0.25
        },
        "2,7",
        "5,7",
        "9,7",
        {
          "x": 1,
          "c": "#cccccc"
        },
        "3,6\n\n\n1,1",
        "6,7\n\n\n1,1"
      ],
      [
        {
          "x": 2.75,
          "c": "#aaaaaa",
          "w": 1.5
        },
        "4,0",
        {
          "c": "#cccccc"
        },
        "5,0",
        "4,1",
        "5,1",
        "4,2",
        "5,2",
        "4,3",
        "5,3",
        "4,4",
        "5,4",
        "4,5",
        "5,5",
        "4,6",
        {
          "w": 1.5
        },
        "5,6",
        {
          "x": 0.25,
          "c": "#aaaaaa"
        },
        "4,7",
        "7,7",
        "11,7"
      ],
      [
        {
          "x": 2.75,
          "w": 1.75
        },
        "6,0",
        {
          "c": "#cccccc"
        },
        "7,0",
        "6,1",
        "7,1",
        "6,2",
        "7,2",
        "6,3",
        "7,3",
        "6,4",
        "7,4",
        "6,5",
        "7,5",
        {
          "c": "#aaaaaa",
          "w": 2.25
        },
        "7,6"
      ],
      [
        {
          "x": 2.75,
          "w": 2.25
        },
        "8,0",
        {
          "c": "#cccccc"
        },
        "8,1",
        "9,1",
        "8,2",
        "9,2",
        "8,3",
        "9,3",
        "8,4",
        "9,4",
        "8,5",
        "9,5",
        {
          "c": "#aaaaaa",
          "w": 2.75
        },
        "8,6\n\n\n2,0",
        {
          "x": 1.25
        },
        "8,7",
        {
          "x": 1.25,
          "c": "#cccccc",
          "w": 1.75
        },
        "8,6\n\n\n2,1",
        "9,6\n\n\n2,1"
      ],
      [
        {
          "x": 2.75,
          "c": "#aaaaaa",
          "w": 1.5
        },
        "10,0",
        "11,1\n\n\n3,0",
        {
          "w": 1.5
        },
        "10,1",
        {
          "c": "#cccccc",
          "w": 7
        },
        "10,3\n\n\n4,0",
        {
          "c": "#aaaaaa",
          "w": 1.5
        },
        "11,4",
        "10,5\n\n\n3,0",
        {
          "w": 1.5
        },
        "11,5",
        {
          "x": 0.25
        },
        "10,6",
        "11,6",
        "10,7"
      ],
      [
        {
          "x": 4.25,
          "d": true
        },
        "11,1\n\n\n3,1",
        {
          "x": 1.5,
          "c": "#cccccc",
          "w": 3
        },
        "11,2\n\n\n4,1",
        "10,3\n\n\n4,1",
        {
          "w": 3
        },
        "11,3\n\n\n4,1",
        {
          "x": 1.5,
          "c": "#aaaaaa",
          "d": true
        },
        "10,5\n\n\n3,1"
      ]
    ]
  }
}

```

Reply:
There's also the VID and PID to be checked for duplicates.